Motorcycle.js

```diff
@@ -1,6 +1,20 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle() {
+	Car.call(this);
+	this.seats = 2;
```

Comment:
You can use an override here inside of an if statement to change the number of seats to 2 if the user enters in a value greater than 2:
```
if (this.seats > 2) {
    this.seats = 2;
}
```